assets/app/views/modals/confirm-replace.html

```diff
@@ -0,0 +1,23 @@
+<div class="modal-resource-update">
+  <div class="modal-body">
+    <div ng-if="resourceList.length === 1">
+      <h1>{{resourceKind | humanizeKind}} '<strong>{{resourceName}}</strong>' already exists</h1>
+      <p>Do you want to replace the current '<strong>{{resourceName}}</strong>' {{resourceKind | humanizeKind}} with the submitted one?</p>
+    </div>
+    
+    <div ng-if="resourceList.length > 1">
+      <h1>Following items from the {{resourceKind | humanizeKind}} already exist:</h1>
```

Comment:
Well I think it's quite handy to know what the user is committing to, in terms what resources are going to be updated, to avoid any problems with overwriting unwanted resource spec